packages/react/react/src/hooks/useForceUpdate.ts

```diff
@@ -7,10 +7,10 @@ const updater = (num: number): number => (num + 1) % 1_000_000;
  * @description
  * 반환된 함수를 실행 시 강제로 리렌더가 실행됩니다.
  *
- * @note useReduce가 state보다 조금 더 저렴(?) 하다고 합니다. https://github.com/streamich/react-use/pull/837
+ * @note useReduce가 state보다 가볍다는 의견이 있습니다. https://github.com/streamich/react-use/pull/837
  */
-export function useForceUpdate() {
+export function useForceUpdate(): () => void {
   const [, forceUpdate] = useReducer(updater, 0);
 
-  return forceUpdate as () => void;
+  return forceUpdate;
```

Comment:
1. I guessed maintainer want to use meaning([Type Assertion](https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#type-assertions)) of `as` to override type React.DispatchWithoutAction (ReturnType<typeof useReducer>[1]) for user

2. and also if use `as`, overriding type can be occured relative code than `: () => void`

Reply:
The reason for the PR I posted is as follows.

1. Override to () => void without Type Assertions.
2. "Type Assertions" forces type information on the type system and skips the type check. Override's role is possible without a "Type Assertions"

-> I decided that there was no reason to choose such a powerful "Type Assertions"